_tests/apps_test.go

```diff
@@ -1,26 +1,32 @@
 package _tests_test
 
 import (
+	"fmt"
 	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/config"
 	. "github.com/onsi/gomega"
 )
 
+func getRandAppName() string {
+	return fmt.Sprintf("apps-test-%d", GinkgoConfig.RandomSeed)
```

Comment:
we could use this as a seed to a `rand` generator, for example:

```
func init() {
    rand.Seed(GinkgoConfig.RandomSeed)
}

var letterRunes = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")

func RandStringRunes(n int) string {
    b := make([]rune, n)
    for i := range b {
        b[i] = letterRunes[rand.Intn(len(letterRunes))]
    }
    return string(b)
}
```

Then we can just use `fmt.Sprintf("apps-test-%d", RandStringRunes(5))`.